src/context/mod.rs

```diff
@@ -163,3 +163,41 @@ macro_rules! context_map {
             .map(|_| context)
     }};
 }
+
+/// Context with all f64::consts available by default.
+/// Alternatively, specifiy constants with math_consts_context!(E, PI, TAU, ...)
```

Comment:
It would be nice to have the code in backticks. Same in line 167.
```suggestion
/// Alternatively, specifiy constants with `math_consts_context!(E, PI, TAU, ...)`
```